### PR TITLE
feat: read description from md files

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

- Add description markdown files
- Read description from the markdown files
- Filter token list to only include tokens that have custom description (markdown file)

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , great. 
Some issues:
1. Section's name font size is smaller in comparison with its subsection
image

2. there is no intend between the subsection name and the text 
image

3. chart lines are in red while should be in green 
image

the same is with 0 (show coloured line instead of the grey line)
image

**Comment by elena-zh** [comment]
Hey @fairlighteth , is this OK to you to have a blue line everywhere in the chart instead of the red/green? 
image

Also, @nenadV91 , @fairlighteth , I think that all section headers should have the same font size. Correct? 
image

Thanks!

File: util/formatUSDPrice.ts [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+  NFTTokenFloorPrice = 'nft-token-floor-price',
+
+  // nft collection stats like number of items, holder, and sales
+  NFTCollectionStats = 'nft-collection-stats',
+
+  // nft floor price with trailing zeros
+  NFTTokenFloorPriceTrailingZeros = 'nft-token-floor-price-trailing-zeros',
+}
+
+const TYPE_TO_FORMATTER_RULES = {
```
Comment (anxolin) [inline util/formatUSDPrice.ts:276]: @nenadV91 just curious, where did you get this logic from?  You did it, or is C&P from somewhere?

Comment (nenadV91) [reply util/formatUSDPrice.ts:276]: Its from Uniswap, the package didn't work on Friday so I created this file. They have a pending PR to fix the issue but its not merged yet https://github.com/Uniswap/conedison/pull/18

**Comment by nenadV91** [comment]
@elena-zh I'm merging this as is because we want to focus on TWAP orders.

**Pull request merged**